exporters/Makefile

```diff
@@ -0,0 +1,58 @@
+PWD                   = $(shell pwd)
+NODE_EXPORTER_VERSION = $(or $(shell printenv NODE_EXPORTER_VERSION), "")
+KUBE_STATE_METRICS    = $(or $(shell printenv KUBE_STATE_METRICS), "")
+
+
+.PHONY: build
+build: kube-state-metrics prometheus-node-exporter
+
+.PHONY: kube-state-metrics
+kube-state-metrics: docker-kustomize-kube-state-metrics docker-vanilla-kube-state-metrics
+
+.PHONY: kustomize-kube-state-metrics
+kustomize-kube-state-metrics: kube-state-metrics-template kube-state-metrics-kustomization.yaml
+
+.PHONY: kube-state-metrics-template
+kube-state-metrics-template:
+	helm template kube-state-metrics stable/kube-state-metrics --version=${KUBE_STATE_METRICS} --output-dir .
+
+.PHONY: vanilla-kube-state-metrics
+vanilla-kube-state-metrics:
+	@mkdir -p vanillas
+	helm template kube-state-metrics stable/kube-state-metrics --version=${KUBE_STATE_METRICS} > vanillas/vanilla-kube-state-metrics.yaml
+
+.PHONY: prometheus-node-exporter
+prometheus-node-exporter: docker-kustomize-prometheus-node-exporter docker-vanilla-prometheus-node-exporter
+
+.PHONY: kustomize-prometheus-node-exporter
+kustomize-prometheus-node-exporter: prometheus-node-exporter-template prometheus-node-exporter-kustomization.yaml
+
+.PHONY: prometheus-node-exporter-template
+prometheus-node-exporter-template:
+	helm template prometheus-node-exporter stable/prometheus-node-exporter --version=${NODE_EXPORTER_VERSION} --output-dir .
+
+.PHONY: vanilla-prometheus-node-exporter
+vanilla-prometheus-node-exporter:
+	@mkdir -p vanillas
+	helm template prometheus-node-exporter stable/prometheus-node-exporter --version=${NODE_EXPORTER_VERSION} > vanillas/vanilla-prometheus-node-exporter.yaml
+
+.SECONDEXPANSION:
+%-kustomization.yaml: $$(wildcard $$*/templates/*.yaml)
+	@echo "kustomization $^"
+	cd $* && ../kustomization.yaml.sh $$(ls templates/*) > kustomization.yaml
```

Comment:
I changed the target to be the actual file, but the problem is that make the dependencies will be always newer than the target because we have nothing as dependencies for the other manifests templating.

Reply:
> I changed the target to be the actual file, but the problem is that make the dependencies will be always newer than the target because we have nothing as dependencies for the other manifests templating.

That shouldn't be, we have a file path with a wildcard, and some dependencies using globbing, but they're all concrete files, and thus should have different, and comparable file modification/creation timestamps. Is there something I'm missing?

Reply:
Never mind, I missed that this is a super old comment :see_no_evil: